front/src/components/LoginPanel/comp/RegisterForm.js

```diff
@@ -47,11 +47,20 @@ const RegisterForm = (props) => {
 
     const handleSubmit = (event) => {
         event.preventDefault();
-        if (
-            /^[a-zA-Z0-9]+@[a-zA-Z0-9]+\.[A-Za-z]+$/.test(email) &&
-            password1 &&
-            password2 === password1
-        ) {
+        if (!login) {
+            document.getElementById("error-msg").innerHTML = "Podaj login";
+        } else if (!/^[a-zA-Z0-9]+@[a-zA-Z0-9]+\.[A-Za-z]+$/.test(email)) {
+            document.getElementById("error-msg").innerHTML = "Błędny email";
```

Comment:
You take e-mail as input, validate it and do nothing with it :) Backend now not only accepts, but **requires e-mail**. Please, check if your component works before **every** commit. Register attempt returns:
```
{
   "message":"Field \"register\" argument \"email\" of type \"String!\" is required, but it was not provided."
}
```